cmd/goal/account.go

```diff
@@ -869,11 +896,16 @@ var addParticipationKeyCmd = &cobra.Command{
 		// Generate a participation keys database and install it
 		client := ensureFullClient(dataDir)
 
-		_, _, err := client.GenParticipationKeysTo(accountAddress, roundFirstValid, roundLastValid, keyDilution, partKeyOutDir)
-		if err != nil {
-			reportErrorf(errorRequestFail, err)
+		participationGen := func() {
+			_, _, err := client.GenParticipationKeysTo(accountAddress, roundFirstValid, roundLastValid, keyDilution, partKeyOutDir)
+			if err != nil {
+				reportErrorf(errorRequestFail, err)
+			}
+			fmt.Println("Participation key generation successful")
```

Comment:
try to follow the goal utility conventions of defining output strings in messages.go, and printing these using reportInfof() instead of writing directly to stdout.